### PR TITLE
Quiz questions Page

### PR DESCRIPTION
**Pull request by Ankur9669**
image

### CODE REVIEW

File: common.css [hunk 1]
```diff
@@ -86,6 +89,27 @@ span{
     display: grid;
     grid-template-columns: 300px auto;
```
Comment (kaustubh-jsr) [inline common.css]: Is there a reason you used px instead of rem?

File: common.css [hunk 2]
```diff
@@ -86,6 +89,27 @@ span{
...
+.quiz-heading-container{
+    margin: 2rem 2rem;
+}
+.quiz-btn-container{
+    margin: 2rem 2rem;
+}
+.quiz-info-container{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    max-width: 1200px;
```
Comment (kaustubh-jsr) [inline common.css]: You can use rem instead of px here too.

File: quizinstruction.html [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+    <link rel="stylesheet" href="quizinstruction.css" />
+  </head>
+
+  <body>
+    <!--  ******************************** Navbar Start ********************************  -->
+    <nav class="navbar">
+      <div>
+        <img
+          alt="hamburger-icon"
+          src="../images/hamburger.svg"
```
Comment (kaustubh-jsr) [inline quizinstruction.html]: I think this image is not loading. Please check and rectify.

File: common.css (hunk 2 above)
Comment (Ankur9669) [reply common.css]: Here i dont want to change its width according to font size

File: quizinstruction.html (hunk 3 above)
Comment (Ankur9669) [reply quizinstruction.html]: Ok i will see this

File: common.css (hunk 1 above)
Comment (Ankur9669) [reply common.css]: I will change it to rems

File: common.css (hunk 2 above)
Comment (kaustubh-jsr) [reply common.css]: > Here i dont want to change its width according to font size

Good to know. Rest looks good to merge.

File: quizinstruction.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+@media only screen and (max-width: 1080px){
+    .hero-section-container{
+        grid-column: 1 / 3;
+    }
+    .sidebar{
+        display: none;
+    }
+}
+@media only screen and (max-width: 750px){
```
Comment (yaswanthmyneni) [inline quizinstruction.css]: suggestion: you can use CSS variables for media queries.

File: quizquestions.html [hunk 5]
```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
```
Comment (yaswanthmyneni) [inline quizquestions.html:1]: UI improvement: you can remove "read instructions carefully" in quiz questions.
Screenshot 2022-02-20 at 2 31 01 PM

**Review by yaswanthmyneni — APPROVED** [review]
You can merge it.

File: common.css [hunk 6]
```diff
@@ -84,11 +87,75 @@ span{
...
+.quiz-heading-container .image-container{
+    display: none;
+}
+.quiz-info-container{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    max-width: 75rem;
+}
+.quiz-info-container .image-container{
+    margin: 0rem 2rem;
```
Comment (pradipgharge) [inline common.css:117]: need not put unit with 0, it still works.

File: common.css [hunk 7]
```diff
@@ -84,11 +87,75 @@ span{
...
+    max-height: 300px;
+    max-width: 400px;
```
Comment (pradipgharge) [inline common.css:119]: use rems if possible.

**Review by pradipgharge — APPROVED** [review]
The code looks good to merge!

File: quizquestions.html (hunk 5 above)
Comment (Ankur9669) [reply quizquestions.html:1]: But why to remove it😅

File: quizinstruction.css (hunk 4 above)
Comment (Ankur9669) [reply quizinstruction.css]: Yes i can use it but i wanna keep as this way for now

File: common.css (hunk 7 above)
Comment (Ankur9669) [reply common.css:119]: Ya but i dont wanna change its width on the basis of font size

**Pull request merged**